Source/DataSources/PolylineGeometryUpdater.js

```diff
@@ -446,12 +453,8 @@ define([
         this._primitives = primitives;
         this._geometryUpdater = geometryUpdater;
         this._positions = [];
-    };
 
-    var generateCartesianArcOptions = {
-        positions : undefined,
-        granularity : undefined,
-        height : undefined
+        generateCartesianArcOptions.ellipsoid = geometryUpdater._scene.globe.ellipsoid;
```

Comment:
`scene.globe` is not guaranteed to exist, generally we prefer `scene.mapProjection.ellipsoid` for this kind of thing.